recruiter_portal_api/settings.py

```diff
@@ -90,9 +90,13 @@
 # https://docs.djangoproject.com/en/4.2/ref/settings/#databases
 
 DATABASES = {
-    'default': {
-        'ENGINE': 'django.db.backends.sqlite3',
-        'NAME': BASE_DIR / 'db.sqlite3',
+      'default': {
```

Comment:
Can we fetch these details from .env? Because we might have different credentials and it may create merge conflicts

Reply:
Sure! Will do